webpack/components/ErratumTypeLabel.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { BugIcon, EnhancementIcon, SecurityIcon, CubeIcon } from '@patternfly/react-icons';
+
+const ErratumTypeLabel = ({ type }) => {
+  switch (type) {
+    case 'bugfix':
+      return (
+        <p><BugIcon /> Bugfix</p>
+      );
+    case 'enhancement':
+      return (
+        <p><EnhancementIcon /> Enhancement</p>
+      );
+    case 'security':
+      return (
+        <p><SecurityIcon /> Security</p>
+      );
+    default:
+      return (
+        <p><CubeIcon /> {type}</p>
```

Comment:
Thoughts on using UnknownIcon instead of Cube..We use Cube to denote Component CVs so might be confusing.

Reply:
Sounds fine to me. I thought I saw another spot that used the Cube for unknown.